thirdparty/cnpy/cnpy.h

```diff
@@ -1,294 +1,353 @@
-//Copyright (C) 2011  Carl Rogers
-//Released under MIT License
-//license available in LICENSE file, or at http://www.opensource.org/licenses/mit-license.php
+// Copyright (C) 2011  Carl Rogers
```

Comment:
Could you please revert code style changes?

Reply:
done